examples/test_record/test_recorder_example.py

```diff
@@ -0,0 +1,94 @@
+##########################################################################
+# Copyright (c) 2010-2021 Robert Bosch GmbH
```

Comment:
# Copyright (c) 2010-2022 Robert Bosch GmbH
to be updated every where (VS code could help you there ;) )